import_flow_events.py

```diff
@@ -32,17 +32,22 @@
 EVENTS_PREFIX = "fxa-flow/data/"
 EVENTS_FILE_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX + "flow-{day}.csv"
 
-# We import each into is own table and maintain a UNION ALL view over them.
+# There are three tables:
+#   * flow_csv      - raw data from the CSV file
```

Comment:
One nit, I wonder if we can name this table so that it's more obviously a temporary import table.  `raw_flow_data_import` or something like that?  No biggie but something to think about.